src/celeritas/inp/Surfaces.hh

```diff
@@ -0,0 +1,251 @@
+//------------------------------- -*- C++ -*- -------------------------------//
+// Copyright Celeritas contributors: see top-level COPYRIGHT file for details
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file celeritas/inp/Surfaces.hh
```

Comment:
Rename `SurfacePhysics` ; it's not how surfaces are defined (geometry) but physics (what happens on them)